alkira/resource_alkira_credential_pan.go

```diff
@@ -67,6 +69,7 @@ func resourceCredentialPanRead(d *schema.ResourceData, meta interface{}) error {
 }
 
 func resourceCredentialPanUpdate(d *schema.ResourceData, meta interface{}) error {
+	return nil
```

Comment:
Once you do this, you could remove all left codes in the function :) but it's okey.